### PR TITLE
(partial) fix Travis CI#1339

### PR DESCRIPTION
**Pull request by agebhar1**
> Issues :-(
> Links 
>  1. [L285] 301 https://github.com/thecodeteam/rexray  → https://github.com/rexray/rexray 
>  2. [L327] 404 https://github.com/atcol  
>  3. [L327] 301 https://github.com/atcol/docker-registry-ui  → https://github.com/squidnyan/docker-registry-ui 
>  4. [L411] 404 http://www.projectatomic.io/registry/  
>  5. [L414]  https://cycle.io/ Cannot assign requested address - connect(2) for "cycle.io" port 443 
>  6. [L510] 301 https://github.com/gaia-adm/pumba  → https://github.com/alexei-led/pumba 
>  7. [L610] 503 http://docs.projectatomic.io/container-best-practices/  
>
> Dupes 
>  None ✓

Strange, did not found infos for Atomic Registry any more. L610 and L411 are not fixed within this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -412,7 +412,7 @@ Services to securely store your Docker images.
 * [Atomic Registry](http://www.projectatomic.io/registry/) - Red Hat Atomic Registry is an open source enterprise registry based on the Origin and Cockpit projects, enhancing the Docker registry library.
 * [Azure Container Registry](https://azure.microsoft.com/de-de/services/container-registry/) Manage a Docker private registry as a first-class Azure resource
 * [CargoOS](https://github.com/RedCoolBeans/cargos-buildroot) - A bare essential OS for running the Docker Engine on bare metal or Cloud. By [@RedCoolBeans](https://github.com/RedCoolBeans)
-* [Cycle.io](https://cycle.io/) Bare-metal container hosting.
+* [Cycle.io](http://cycle.io/) Bare-metal container hosting.
```
Comment (veggiemonk) [inline README.md]: I tried this URL and it redirects automatically to https....

**Comment by veggiemonk** [comment]
@agebhar1 great job! Thanks a lot. 
Just one minor change

File: README.md (hunk 1 above)
Comment (agebhar1) [reply README.md]: ... I'll drop this commit

**Comment by agebhar1** [comment]
You're welcome @veggiemonk. I dropped the commit for »Cycle«. What's your opinion about »Atomic Registry«?

**Comment by veggiemonk** [comment]
@agebhar1 I’ll investigate for project atomic because I feel it’s been killed.

**Comment by vegasbrianc** [comment]
@agebhar1 @veggiemonk Since Red Hat recently bought CoreOS it makes senses that they migrate everything towards Quay. I see RH documentation slowly indicating this as well

**Comment by agebhar1** [comment]
@vegasbrianc thank you for the update, I didn't found any hints on Atomics website.

**Pull request merged**